wordpress-mobile/AztecEditor-iOS
# Unhooks the DOM synchronization.

Addresses one of the steps in #492.

**Testing:**

- Use the editor to write anything
- Use any styles.
- Switching to HTML mode should show an empty document.
